.github/workflows/build.yml

```
@@ -23,7 +23,7 @@ jobs:
           windows-latest,
           ubuntu-latest
         ]
-        java-version: [17, 18]
+        java-version: [17, 18, 22]
```

Consider:
```suggestion
        java-version: [17, 21, 22]
```

So that it definitely works on LTS 21. Testing only on 22 could result in the library using 22-only features that are not in 21.